lib/TerminalDisplay.cpp

```diff
@@ -340,6 +363,10 @@ TerminalDisplay::TerminalDisplay(QWidget *parent)
 ,_cursorShape(BlockCursor)
 ,mMotionAfterPasting(NoMoveScreenWindow)
 {
+  // variables for draw text
```

Comment:
Keep with the pattern set above and use the constructor member initializer syntax for these initializations.